client/public/elements/pages/concept/rp-page-concept.tpl.js

```diff
@@ -224,16 +224,16 @@ return html`
               </rp-person-preview>
             `:html``}
 
-          `  
-          )}   
-          ${ this.tempResearch.length > 10 ? 
+          `
+          )}
+          ${ this.tempResearch.length > 8 ?
```

Comment:
@sbagg.  Needed to go back to 8 since the current search is designed to fetch 10 max, and itwould not show the more box.